src/main/java/com/epam/esm/model/Certificate.java

```diff
@@ -0,0 +1,80 @@
+package com.epam.esm.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.util.Set;
+
+public class Certificate extends Entity implements Serializable {
+
+   private String  name;
+   private String description;
+   private Double price;
+   private Integer duration;
+   @JsonFormat(shape = JsonFormat.Shape.STRING)
+   private Timestamp createDate = new Timestamp(System.currentTimeMillis());
```

Comment:
It's better to use LocalDateTime or Instant.
https://javadevblog.com/polnoe-rukovodstvo-po-java-8-date-time-api-primery-localdate-instant-localdatetime-parse-i-format.html

Reply:
Changed to Instant